topics/assembly/tutorials/assembly-quality-control/slides.html

```diff
@@ -0,0 +1,316 @@
+- **QUAST-LG**: for large genomes (e.g., mammalians).
+- **rnaQUAST**: for RNAseq.
+- **Icarus**: an interactive visualizer for these tools.
+
+
+It also includes:
+- Reads mapping (mi-assemblies evaluation).
+- Kmer representation (KMC)
+- Structural prediction modules (GeneMark,  GlimmerHMM, Barrnap and BUSCO).
+- For metagnomics dataset: MetaGeneMark, Krona tools, BLAST, and SILVA 16S rRNA database.
+
+---
+
+# **C**ompleteness
+
+---
+
+### Assembly size vs estimated
+
+Proportion of the original genome represented by the assembly:
+<br>
+
+.image-100[
+![Formula to estimate assembly size completeness](../../images/genome-qc/assembly-size-qc-formula.png)
+]
+
+<br>
+- "*" it’s an estimation, so not perfect
+
+---
+
+### Core genes
+
+Quantitative assessment of genome assembly based on evolutionarily informed expectations of gene content from near-universal single-copy orthologs.
+
+.pull-left[
+.image-70[
+![Formula to estimate assembly completeness for core genes](../../images/genome-qc/busco-formula.png)
+]
+]
+.pull-right[
+.image-60[
+![Example of BUSCO plot for Nosema species (Microsporidia) ](../../images/genome-qc/BUSCO-scores.png)
+]]
+
+Tips: Reference databases are constructed using known genomes. Species with few/no close genomes available can have very bad scores.
+
+---
+
+### Core genes evalutation software
+
+**BUSCO**: Assessing genome assembly and annotation completeness with **B**enchmarking **U**niversal **S**ingle-**C**opy **O**rthologs
+
+.image-40[
+![Explanation of orthology classification for creation of BUSCO sets](../../images/genome-qc/busco_sampling.png)
+]
+
+.footnote[Waterhouse, R. M., Zdobnov, E. M. & Kriventseva, E. V. Correlating Traits of Gene Retention, Sequence Divergence, Duplicability and Essentiality in Vertebrates, Arthropods, and Fungi. Genome Biol Evol 3, 75–86 (2011).]
+
+---
+
+### BUSCO limitations
+
+**The value of the BUSCO is only as good as its reference database.**
+<br>
+<br>
+Example with BUSCO Eukaryotic set:
+.pull-left[
+.image-70[
+![Limitation of BUSCO orthoDB for eukaOrthoDB - part A](../../images/genome-qc/BUSCO_limitations_A.png)
+]]
+.pull-right[
+.image-70[
+![Limitation of BUSCO orthoDB for eukaOrthoDB - part B](../../images/genome-qc/BUSCO_limitations_B.png)
+]]
+
+.footnote[Saary, P., Mitchell, A. L. & Finn, R. D. Estimating the quality of eukaryotic genomes recovered from metagenomic analysis with EukCC. Genome Biol 21, 244 (2020).]
+
+---
+
+### Assembly kmer content
+
+The aim is to check assembly coherence against the content within reads that were used to produce the assembly. Basically, how many elements of each frequency on the read’s spectrum ended up not included in the assembly, included once, included twice etc.
+<br>
+<br>
+
+- KAT spectra-cn plot
+- Histogram is build with read kmer content.
+- Colors come from assembly:
+  - **Black = not in the assembly** (heterozygous part, second haplotype).
+  - **Red = once in the assembly.**
+
```

Comment:
I suggest you to modify a little bit this section, in order to use [Meryl](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy4) + [Merqury](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/merqury/merqury/1.3+galaxy1) as reference instead of KAT, as the latter is not available in Galaxy. 

You can find some example of Merqury plots in https://github.com/marbl/merqury/tree/master/example

![triocanu_clr cvi spectra-cn fl](https://user-images.githubusercontent.com/26678117/148070369-aabc2d79-aecc-460a-a158-501578bb2195.png)

Reply:
Thanks for the suggestion. I was thinking KAT was available but it's only some functions :/ I've asking to @lleroi if she can makes a wrapper for kat comp as I found it more user friendly and straightforward.

Reply:
I can work on it too if you consider it provides some advantages with respect meryl +  merqury. In any case, I think it would be appropriate to include some reference to alternative tools to KAT. Regards!